docs/4_how_to_search_for_publishers.md

```diff
@@ -0,0 +1,16 @@
+# How to search for publishers
+
+This tutorial will show you how to search for specific publishers in the `PublisherCollection`.
+
+## Using `search()`
+There a quite a few differences between the publishers, especially in the attributes the underlying parser supports.
+You can search through the collection to get only publishers fitting your use case by utilizing the `search()` method.
+
+Let's get some publishers based in the US, supporting an attribute called `topics` and `NewsMap` as a source, and use them to initialize a crawler afterward.
+
+````python
+from fundus import Crawler, PublisherCollection, NewsMap
+
+fitting_publishers = PublisherCollection.us.search(attributes=["topics"], source_types=[NewsMap])
```

Comment:
The end is quite sudden. Maybe include a summary? Some sort of tutorial for getting help? Both of these might be done in separate prs.

Reply:
I also thought about including this in the third tutorial, but yeah, that's something for another PR I'm fine with leaving it as is.